fs/fs.go

```diff
@@ -559,78 +561,70 @@ func (self *RealFsInfo) GetDirFsDevice(dir string) (*DeviceInfo, error) {
 	return nil, fmt.Errorf("could not find device with major: %d, minor: %d in cached partitions map", major, minor)
 }
 
-func (self *RealFsInfo) GetDirDiskUsage(dir string, timeout time.Duration) (uint64, error) {
+func (self *RealFsInfo) GetDirUsage(dir string) (UsageInfo, error) {
 	claimToken()
 	defer releaseToken()
-	return GetDirDiskUsage(dir, timeout)
-}
 
-func GetDirDiskUsage(dir string, timeout time.Duration) (uint64, error) {
```

Comment:
I think we had this function declaration so other binaries can use this function without needing to create a `RealFsInfo`.  Can you add this back in?

Reply:
I have added `GetDirInodeUsage(string, time.Duration) (uint64, error)` and `GetDirDiskUsage(string, time.Duration) (uint64, error)` back in and have marked its as deprecated. Also `GetDirUsage(string) (UsageInfo, error)` was added. I think this make sense according to the `semver`